File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.cpp
class Solution {
public:
    vector<int> topKFrequent(vector<int>& nums, int k) {
        unordered_map<int,int>pp;
        for(auto n:nums){
            pp[n]++;
        }

        priority_queue<pair<int,int>,vector<pair<int,int>>,greater<pair<int,int>>> que;
        
        for(auto it:pp){
            que.push(make_pair(it.second,it.first));
            if(que.size()>k){
                que.pop();
            }
        }
        vector<int>ans;
        while(!que.empty()){
            ans.push_back(que.top().second);
            que.pop(); // missing
        }

        return ans;
    }
};

File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int[] ans=new int[2*nums.length];
        int j=nums.length;
        for(int i=0;i<2*nums.length;i++){
            if(i<nums.length){
                ans[i]=nums[i];
            }
            else{
                ans[i]=nums[i-j];
            }
        }

        return ans;
    }
}